pages/upgrade-guide/3.0.md

```diff
@@ -159,6 +159,10 @@ These properties were deprecated in jQuery 1.9, as they were only used for the o
 
 https://github.com/jquery/jquery/issues/1908
 
+#### Breaking change: Deprecated `.size()` removed
+
+`.size()` is deprecated in jQuery 1.8 and removed in jQuery 3.0 in favor to `.length` property.
```

Comment:
Couple of suggestions:

- is deprecated in -> is deprecated as of
- in favor to -> in favor of the